doc/kci_testsuite.md

```diff
@@ -125,11 +110,18 @@ We will now look into last three config entries, namely: `extra_packages`, `scri
 ```
 $ cat jenkins/debian/debos/script/buster-btrfs.sh
 
-git clone --depth=1 https://github.com/kdave/btrfs-progs.git /btrfs-progs
+#!/bin/bash
+
+BUILD_DEPS="pkg-config git make autoconf automake gcc \
+make libblkid-dev zlib1g-dev liblzo2-dev libzstd-dev e2fslibs-dev python3-setuptools python3.7-dev"
+
+apt-get update -y && apt-get install -y $BUILD_DEPS
+GIT_SSL_NO_VERIFY=1 git clone --depth=1 https://github.com/kdave/btrfs-progs.git /btrfs-progs
 cd /btrfs-progs
-./autogen.sh && ./configure
-make
-make install
+./autogen.sh && ./configure --disable-documentation
```

Comment:
Maybe add `set -e` at the top of the script and drop the `&&` to put only one command on each line.